src/audio/pipeline.c

```diff
@@ -429,7 +430,7 @@ int pipeline_prepare(struct pipeline *p, struct comp_dev *dev)
 	/* initialize task if necessary */
 	if (!p->pipe_task) {
 		type = pipeline_is_timer_driven(p) ? SOF_SCHEDULE_LL_TIMER :
-			SOF_SCHEDULE_EDF;
+			SOF_SCHEDULE_LL_DMA;
```

Comment:
How would we use EDF scheduling then with pipeline ?

Reply:
We don't. Right now pipeline is treated as a low latency component as a whole. Scheduling it as an EDF task doesn't benefit us at all. In the future more complex components will be added (e.g. with processing frames exceeding standard 1 ms), which alone will be scheduled as EDF tasks (cAVS complaint).